mlflow/pytorch.py

```diff
@@ -135,6 +155,10 @@ def save_model(pytorch_model, path, conda_env=None, mlflow_model=Model(), **kwar
                         }
 
     :param mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
+    :param code_paths: A list of local filesystem paths to Python file dependencies (or directories
+                       containing file dependencies). These files will be *prepended* to the system
+                       path when the model is as a generic python function via
```

Comment:
Instead of doing this in a followup PR (see description), I've merged the proposed changes for `load_model` into the current branch. Let me know how they look!

Reply:
Fixed the typo! Thanks for catching it.